_layouts/plain.html

```diff
@@ -5,7 +5,6 @@
 <div class="fill">
   <header>
     <nav class="page-centered">
-      <a class="vl-title" href="{{site.baseurl}}/">{{site.title}}</a>
```

Comment:
Ah. I thought Vega removed it from their main documentation. It's not there on their main page. I will fix it